common/djangoapps/student/admin.py

```diff
@@ -175,14 +175,15 @@ class UserAdmin(BaseUserAdmin):
     """ Admin interface for the User model. """
     inlines = (UserProfileInline,)
 
-    def get_readonly_fields(self, *args, **kwargs):
+    def get_readonly_fields(self, request, obj=None):
         """
         Allows editing the users while skipping the username check, so we can have Unicode username with no problems.
         The username is marked read-only regardless of `ENABLE_UNICODE_USERNAME`, to simplify the bokchoy tests.
```

Comment:
"The username is marked read-only regardless of..." -> "The username is marked read-only when editing existing users regardless of.."

Reply:
@OmarIthawi 
done